sru-templates/manual/azure-sru

```diff
@@ -15,11 +17,42 @@ RESOURCE_GROUP="<specify your resource group>"
 BOOT_DIAG="<specify your boot diagnostics group>"
 SSH_KEY="$HOME/.ssh/id_rsa.pub"
 
+
+IPV6_NET_REGISTERED=`az feature show --name AllowIPv6VirtualNetwork --namespace Microsoft.Network | jq -r '.properties.state'`
+IPV6_CA_LB_REGISTERED=`az feature show --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network | jq -r '.properties.state'`
+
+if [ "$IPV6_NET_REGISTERED" != "Registered" -o "$IPV6_CA_LB_REGISTERED" != "Registered" ]; then
+	echo "Account is not yet registered for AllowIPv6VirtualNetwork or AllowIPv6CAOnStandardLB. Can take 30 mins. Activate feature now (y/n)?"
+    read RESP
+    if [ "$RESP" = "y" ]; then
+        az feature register --name AllowIPv6VirtualNetwork --namespace Microsoft.Network
+        az feature register --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network
+    fi
+    echo "Waiting for AllowIPv6VirtualNetwork freature registration"
+    while [ "$IPV6_NET_REGISTERED" != "Registered" -o "$IPV6_CA_LB_REGISTERED" != "Registered" ]; do
+        echo -n "."
+        sleep 5
+        IPV6_NET_REGISTERED=`az feature show --name AllowIPv6VirtualNetwork --namespace Microsoft.Network | jq -r '.properties.state'`
+        IPV6_CA_LB_REGISTERED=`az feature show --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network | jq -r '.properties.state'`
+    done
+fi
+
+IPV6_REGISTRATION_COMPLETE=`az provider show -n Microsoft.Network | jq -r ".registrationState"`
+echo "Waiting for Microsoft.Network IPv6 feature registration"
+while [ "$IPV6_REGISTRATION_COMPLETE"  != "Registered" ]; do
+    echo -n "."
+    sleep 5;
+    IPV6_REGISTRATION_COMPLETE=`az provider show -n Microsoft.Network | jq -r ".registrationState"`
+done
+
+
 cat > sethostname.yaml <<EOF
 ## template: jinja
 #cloud-config
 ssh_import_id : [%SRU_LP_USER%]
 hostname: SRU-worked-{{v1.cloud_name}}
+runcmd:
+  - cp /run/cloud-init/ds-identify.log /var/log
```

Comment:
Should we just be capturing `/run/cloud-init/`? (Should we just be running `cloud-init collect-logs`?)

Reply:
Actually there is no point in this currently as I'm not actually validating that input anywhere in this SRU. I'll back this out and we can discuss whether we need to capture anything. I had originally had this in there because I lost /run across a reboot and was trying to debug if the datasource actually got detected properly. I should not have reflected this everywhere.